anymail/backends/sendinblue.py

```diff
@@ -0,0 +1,144 @@
+        http_headers['Content-Type'] = 'application/json'
+
+        super(SendinBluePayload, self).__init__(message, defaults, backend, headers=http_headers, *args, **kwargs)
+
+    def get_api_endpoint(self):
+        return "smtp/email"
+
+    def init_payload(self):
+        self.data = {  # becomes json
+            'to': [],
+            'cc': [],
+            'bcc': [],
+            'sender': [],
+        }
+
+    def serialize_data(self):
+        """Performs any necessary serialization on self.data, and returns the result."""
+
+        headers = self.data["headers"]
+        if "Reply-To" in headers:
+            # Reply-To must be in its own param
+            reply_to = headers.pop('Reply-To')
+            self.set_reply_to(parse_address_list([reply_to]))
+        if len(headers) > 0:
+            self.data["headers"] = dict(headers)  # flatten to normal dict for json serialization
+        else:
+            del self.data["headers"]  # don't send empty headers
+
+        return self.serialize_json(self.data)
+
+    #
+    # Payload construction
+    #
+
+    @staticmethod
+    def email_object(email):
+        """Converts EmailAddress to SendinBlue API array"""
+        email_object = dict()
+        email_object['email'] = email.addr_spec
+        if email.display_name:
+            email_object['name'] = email.display_name
+        return email_object
+
+    def set_from_email(self, email):
+        self.data["sender"] = self.email_object(email)
+
+    def set_recipients(self, recipient_type, emails):
+        assert recipient_type in ["to", "cc", "bcc"]
+        if emails:
+            # Normally, exactly one "personalizations" entry for all recipients
+            # (Exception: with merge_data; will be burst apart later.)
+            self.data[recipient_type] = [self.email_object(email) for email in emails]
+            self.all_recipients += emails  # used for backend.parse_recipient_status
+
+    def set_subject(self, subject):
+        if subject != "":  # see note in set_text_body about template rendering
+            self.data["subject"] = subject
+
+    def set_reply_to(self, emails):
+        # SendinBlue only supports a single address in the reply_to API param.
+        if len(emails) > 1:
+            self.unsupported_feature("multiple reply_to addresses")
+        if len(emails) > 0:
+            self.data['replyTo'] = self.email_object(emails[0])
+
+    def set_extra_headers(self, headers):
+        self.data['headers'] = [header for header in headers]
+
+    def set_text_body(self, body):
+        self.data['textContent'] = body
+
+    def set_html_body(self, body):
+        if body is not None:
+            if "htmlContent" in self.data:
+                self.unsupported_feature("multiple html parts")
+
+            self.data['htmlContent'] = body
+
+    def add_attachment(self, attachment):
+        """Converts attachments to SendinBlue API {name, base64} array"""
+
```

Comment:
I can't find any info on SiB support for inline attachments/inline images. (May need to contact them for clarification.) If they truly don't support inlines, we'd need `if attachment.inline:   self.unsupported_feature("inline attachments")` here.

(Also, I'm really surprised they don't have a way to specify the attachment content-type; I'm wondering if they always just guess that from the name. If so, that becomes a "quirks and limitations" entry for the docs.)

Reply:
On the API v2 we have this little part of the documentation.
Maybe the v3 keep it functionnal.. We will need to test i guess.

|Input Parameter | Description|
| -- | -- |
| html | Body of the message (HTML version) [Mandatory]. To send inline images, use < img src="{YourFileName.Extension}" alt="image" border="0" >, the src attribute value inside {} (curly braces) should be same as the filename used in inline_image parameter |
| inline_image | Pass your inline image/s filename & its  base64 encoded chunk data as an associative array. Example:  array(“YourFileName.Extension”=>”Base64EncodedChunkData”). You can  use commas to separate multiple inline images [Optional]. |